cloudformation/start_performance_single.sh

```diff
@@ -0,0 +1,345 @@
+create_stack_command="aws cloudformation create-stack --stack-name $stack_name \
+    --template-body file://singlenode/single-node.yaml --parameters \
+        ParameterKey=AWSAccessKeyId,ParameterValue=$aws_access_key \
+        ParameterKey=AWSAccessKeySecret,ParameterValue=$aws_access_secret \
+        ParameterKey=CertificateName,ParameterValue=$certificate_name \
+        ParameterKey=KeyPairName,ParameterValue=$key_name \
+        ParameterKey=DBUsername,ParameterValue=$db_username \
+        ParameterKey=DBPassword,ParameterValue=$db_password \
+        ParameterKey=WSO2InstanceType,ParameterValue=$wso2_is_instance_type \
+        ParameterKey=BastionInstanceType,ParameterValue=$bastion_instance_type \
+    --capabilities CAPABILITY_IAM"
+
+echo ""
+echo "Creating stack..."
+echo "$create_stack_command"
+stack_id="$($create_stack_command)"
+stack_id=$(echo $stack_id|jq -r .StackId)
+# stack_id="arn:aws:cloudformation:us-east-1:610968236798:stack/is-performance-test-stack-single-node/786da520-359d-11e9-99be-0e94a1882570"
+
+echo ""
+echo "Waiting ${minimum_stack_creation_wait_time}m before polling for cloudformation stack's CREATE_COMPLETE status..."
+sleep ${minimum_stack_creation_wait_time}m
+
+echo ""
+echo "Polling till the stack creation completes..."
+aws cloudformation wait stack-create-complete --stack-name $stack_id
+printf "Stack creation time: %s\n" "$(format_time $(measure_time $stack_create_start_time))"
+
+echo ""
+echo "Getting Bastion Node Public IP..."
+bastion_instance="$(aws cloudformation describe-stack-resources --stack-name $stack_id --logical-resource-id WSO2BastionInstance2 | jq -r '.StackResources[].PhysicalResourceId')"
+bastion_node_ip="$(aws ec2 describe-instances --instance-ids $bastion_instance | jq -r '.Reservations[].Instances[].PublicIpAddress')"
+echo "Bastion Node Public IP: $bastion_node_ip"
+
+
+echo ""
+echo "Getting WSO2 IS Node Private IP..."
+wso2is1_auto_scaling_grp="$(aws cloudformation describe-stack-resources --stack-name $stack_id --logical-resource-id WSO2ISNode1AutoScalingGroup | jq -r '.StackResources[].PhysicalResourceId')"
+wso2is1_instance="$(aws autoscaling describe-auto-scaling-groups --auto-scaling-group-names $wso2is1_auto_scaling_grp | jq -r '.AutoScalingGroups[].Instances[].InstanceId')"
+wso2_is_1_ip="$(aws ec2 describe-instances --instance-ids $wso2is1_instance | jq -r '.Reservations[].Instances[].PrivateIpAddress')"
+echo "WSO2 IS Node Private IP: $wso2_is_1_ip"
+
+echo ""
+echo "Getting RDS Hostname..."
+rds_instance="$(aws cloudformation describe-stack-resources --stack-name $stack_id --logical-resource-id WSO2ISDBInstance2 | jq -r '.StackResources[].PhysicalResourceId')"
+rds_host="$(aws rds describe-db-instances --db-instance-identifier $rds_instance | jq -r '.DBInstances[].Endpoint.Address')"
+echo "RDS Hostname: $rds_host"
+
+copy_isserver_edit_command="scp -i $key_file -o "StrictHostKeyChecking=no" $home/isserver_edit.sh ubuntu@$bastion_node_ip:/home/ubuntu/"
```

Comment:
We don't need to copy $home/setup, isserver_edit.sh, setup_is.sh and createDB.sql files explicitly from here. We're coping the distribution anyway. Please change the logic to take the files from there.

Reply:
Also, let's remove the unnecessary files from this branch and use it for single node performance testing.